source/main/gfx/VideoCamera.cpp

```diff
@@ -237,7 +237,7 @@ void VideoCamera::update(float dt)
 			normal.normalise();
 			Vector3 refx = truck->nodes[nz].AbsPosition - truck->nodes[nref].AbsPosition;
 			refx.normalise();
-			// why does this flip ~2-3� around zero orientation and only with trackercam. back to slower crossproduct calc, a bit slower but better .. sigh
+			// why does this flip ~2-3� around zero orientation and only with trackercam. back to slower crossproduct calc, a bit slower but better .. sigh

```

Comment:
I think the character encoding broke here. Did you perhaps save the files as something else than UTF8?